lib/cli/utils.js

```diff
@@ -88,7 +88,8 @@ utils.iterativeCompletion = (inputs, depth, completer) => {
   // they'll be not valid (those are completions for previous inputs)
   if (newDepth <= inputs.length - 1) completions = [];
   if (!completions[0] ||
-      completions.length === 1 && completions[0] === inputs[depth]) {
+    completions.length === 1 && completions[0] === inputs[depth]
```

Comment:
@lundibundi in your version it's difficult to find code block inside `if`, I want to move `) {` to the next line because at the first sight it's difficult to understand what is `completions = [];` part of condition or inner block.